bin/mail/2017-teacher-application-responses/teacher_application_decision_processor.rb

```diff
@@ -151,80 +151,57 @@ def load_workshop_map
 
   def process(decision, teacher_application, params = {})
     raise "Unexpected decision: #{decision}" unless @results.key? decision
+    params[:decision] = decision
```

Comment:
Altering params passed in is bad practice. You could always do something like
`new_params = params.merge({decision: decision})`